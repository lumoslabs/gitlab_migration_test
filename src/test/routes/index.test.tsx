import * as React from 'react'
import { render } from '@test/render-with-redux'

import { Index } from '@routes/index'


describe('Index page', () => {
  it('matches snapshot', () => {
    const { asFragment } = render(<Index />)
    expect(asFragment()).toMatchSnapshot()
  })
})
