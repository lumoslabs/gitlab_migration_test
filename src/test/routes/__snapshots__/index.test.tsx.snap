// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index page matches snapshot 1`] = `
<DocumentFragment>
  <main>
    <div
      class="app_1mrdeqi"
    >
      <div
        class="mainContent_lvw97s"
      >
        <div
          class="flexColumnJustifyCenter_66qnx8-o_O-fullHeight_149btvt"
        >
          <div
            class="loader"
          />
        </div>
      </div>
    </div>
  </main>
</DocumentFragment>
`;
